helm/mimoto/values.yaml

```diff
@@ -465,5 +465,5 @@ enable_insecure: false
 volumes:
   secrets:
     mimoto:
-      key: keystorep12
+      key: oidckeystore.p12
```

Comment:
is extension is also required

Reply:
The extension is better.easier to understand.